webpack.config.js

```diff
@@ -270,7 +270,7 @@ module.exports = {
 		new DependencyExtractionWebpackPlugin( { injectPolyfill: true } ),
 	].filter( Boolean ),
 	watchOptions: {
-		ignored: '!packages/*/build-module/**/*',
+		ignored: /^(?!.*\/packages\/.*\/build-module\/.*).*$/,
```

Comment:
The regexes in this file are complicated enough that we may benefit from adding test cases to make sure we're matching the expected cases.

Besides the SCSS behavior, @gziolo was this one ready for another review or were we waiting for additional changes?

Reply:
Yes, the only task left is to ensure that changes in SCSS files trigger updates in the top-level `build` folder. Otherwise, everything works as expected.

I'm tempted to change the list of ignored files to `node_modules` 😄

Reply:
Maybe something like /^(?!.*\/packages\/.*\/(build-module|build-style)\/.*).*$/ is going to work?

Reply:
I tried that but it didn't work but I didn't explore it further. In the online RegExp matcher, it was working correctly.

Reply:
Could we try and split this out to multiple glob patterns instead of a single complicated regex?

https://webpack.js.org/configuration/watch/
<img width="798" alt="Screen Shot 2021-03-31 at 10 43 08 AM" src="https://user-images.githubusercontent.com/1270189/113187643-f0f8cf00-920d-11eb-987e-a2ad6c64207c.png">

Reply:
Yes, let's try a simpler approach that ignores the most popular path rather than trying to make it too optimized 😄

Reply:
This works great:
```js
ignored: [ '**/node_modules', '**/packages/*/src' ],
```